modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingBaseScheduleView.js

```diff
@@ -77,13 +77,17 @@ class ChangeTrackingBaseScheduleView extends React.Component {
 			publishTime: publishDate.getTime(),
 		});
 
+		this.setState({setScheduleButtonDisabled: true});
+
 		fetch(portletURL, {
 			method: 'GET',
 		})
 			.then((response) => response.json())
 			.then((json) => {
 				if (json.redirect) {
 					navigate(json.redirect);
+
+					this.setState({setScheduleButtonDisabled: false});
```

Comment:
I think this is not needed since it redirects to another page. It doesn't matter if the button is re-enabled or not.

And do we need to re-enable the button when there is an error? Can you test it by updating the resource command to always return an error?